odata-csdl/7 Structural Property.md

```diff
@@ -155,10 +155,10 @@ value
 
 ## ##subsec Type Facets
 
-Facets modify or constrain the acceptable values of a property.
+Facets modify or constrain the acceptable values for a model element of a given type.
 
-For single-valued properties the facets apply to the value of the
-property. For collection-valued properties the facets apply to the items
+For single-valued model elements the facets apply to the value of the
```

Comment:
As this is within the definition of a property, it seems we can be clearer by using "property" here.